src/ShallowWrapper.js

```diff
@@ -454,6 +456,32 @@ class ShallowWrapper {
   }
 
   /**
+   * Finds ref of the current wrapper.
+   *
+   * NOTE: can only be called on a wrapper of a single node.
+   *
+   * @param {String} ref
+   * @returns {ShallowWrapper}
+   */
+  ref(ref) {
+    return this.single('ref', (node) => {
+      if (!isAReactComponentNode(this.unrendered)) {
+        throw new Error('The current node must be a React component.');
+      }
+
+      const result = findRefInOwnedChildren(node, ref, true);
+
+      if (result.length === 0) return false;
+      if (result.length === 1) return new ShallowWrapper(result[0], this.root);
+      if (result.length > 1) {
+        throw new Error('There is more than one component own by the tree root with the same ref string');
+      }
+
+      return false;
```

Comment:
Oupsi... you meant the `return false`. I thought the thrown error. You are right again !